test/files/neg/java-annotation-bad.check

```diff
@@ -1,13 +1,13 @@
-Test_1.scala:12: error: Java annotation Ann_0 is abstract; cannot be instantiated
+Test_1.scala:12: error: Java annotation Ann_0 is trait; cannot be instantiated
```

Comment:
Java doesn't have traits. I think Lukas commented that he preferred "abstract"?

Reply:
> I think Lukas commented that he preferred "abstract"?

@hrhino mentioned he prefered it mention trait and I do as well, but I could change it back.